core/src/main/scala/com.snowplowanalytics.stream/loader/Emitter.scala

```diff
@@ -0,0 +1,190 @@
+/**
+  * Copyright (c) 2014-2017 Snowplow Analytics Ltd.
+  * All rights reserved.
+  *
+  * This program is licensed to you under the Apache License Version 2.0,
+  * and you may not use this file except in compliance with the Apache
+  * License Version 2.0.
+  * You may obtain a copy of the Apache License Version 2.0 at
+  * http://www.apache.org/licenses/LICENSE-2.0.
+  *
+  * Unless required by applicable law or agreed to in writing,
+  * software distributed under the Apache License Version 2.0 is distributed
+  * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+  * either express or implied.
+  *
+  * See the Apache License Version 2.0 for the specific language
+  * governing permissions and limitations there under.
+  */
+
+package com.snowplowanalytics.stream.loader
+
+
+import clients.BulkSender
+import com.amazonaws.services.kinesis.connectors.UnmodifiableBuffer
+import com.amazonaws.services.kinesis.connectors.interfaces.IEmitter
+
+import scala.collection.mutable.{Buffer => SMBuffer}
+
+
+// Java
+import java.io.IOException
+import java.util.{List => List}
+
+// Scalaz
+import scalaz._
+import Scalaz._
+
+// Scala
+import scala.collection.mutable.ListBuffer
+
+import scala.collection.immutable.{List => SList}
+import scala.collection.JavaConverters._
+
+
+import sinks.ISink
```

Comment:
imports need to be sorted, you can look at the existing code for guidelines

Reply:
addressed